content/SCALE/SCALETutorials/Apps/CommunityApps/jellyfin.md

```diff
@@ -0,0 +1,172 @@
+
+Jellyfin also requires a dataset or emptyDir for **Jellyfin Transcodes Storage**.
+Select **ixVolume (dataset created automatically by the system)** in **Type** to let SCALE create the dataset, select **Host Path** to use an existing dataset created on the system, or select **emptyDir** to use a temporary directory on the disk or in memory.
+
+Accept the defaults in Resources or change the CPU and memory limits to suit your use case.
+
+Click **Install**.
+
+A container is launched with root privileges to apply the correct permissions to the Jellyfin directories.
+Afterward, the Jellyfin container runs as a non-root user (Default: 568).
+Configured storage directories are chowned if the parent directory does not match the configured user.
+
+The system opens the **Installed Applications** screen with the Jellyfin app in the **Deploying** state.
+When the installation completes it changes to **Running**.
+
+{{< trueimage src="/images/SCALE/Apps/JellyfinInstalled.png" alt="Jellyfin Installed" id="Jellyfin Installed" >}}
+
+Click **Web Portal** on the **Application Info** widget to open the Jellyfin web interface initial setup wizard to set up your admin account and begin administering libraries.
+
+{{< trueimage src="/images/SCALE/Apps/JellyfinWebPortal.png" alt="Jellyfin Web Portal" id="Jellyfin Web Portal" >}}
+
+## Editing the Jellyfin Application
+
+Go to the **Installed Applications** screen and select Jellyfin from the list of installed applications.
+Click **Edit** on the **Application Info** widget to open the **Edit Jellyfin** screen.
+The settings on the edit screen are the same as on the install screen.
+You cannot edit **Storage Configuration** paths after the initial app install.
+
+Click **Update** to save changes.
+TrueNAS automatically updates, recreates, and redeploys the Jellyfin container with the updated environment variables.
+
+## Understanding Jellyfin Settings
+
+The following sections provide more detailed explanations of the settings found in each section of the **Install Jellyfin** screen.
+
+### Application Name Settings
+
+Accept the default value or enter a name in the **Application Name** field.
+In most cases use the default name, but if adding a second deployment of the application you must change this name.
+
+Accept the default version number in **Version**.
+When a new version becomes available, the application has an update badge.
+The **Installed Applications** screen shows the option to update applications.
+
+### Jellyfin Configuration Settings
+
+You can accept the defaults in the **Jellyfin Configuration** settings, or enter the settings you want to use.
+
+{{< trueimage src="/images/SCALE/Apps/InstallJellyfinConfigSettings.png" alt="Jellyfin Configuration Settings" id="Jellyfin Configuration Settings" >}}
+
+You can enter a **Published Server URL** for use in UDP autodiscovery, or leave it blank.
+
+If needed, click **Add** to define **Additional Environment Variables**.
+
+### User and Group Settings
+
```

Comment:
Suggestion. Add something in the First Steps section about creating a new user and group. They need this before they begin the app install if they want to use something other than the default setting. Just something to consider.